### ISSUE TITLE
Spark Double enter without counters if copying a copy

### ISSUE THREAD

**Issue by copperfield42** [magefree/mage#9661]
![image](https://user-images.githubusercontent.com/4504719/196017832-c1debf92-72c9-4d31-a191-821618c3dc4f.png)

I have a [[Spark Double]] (bottom) enter as a copy of [[Phyrexian Metamorph]] (top) that in turn is as copy of an opponent [[Serra Ascendant]], and it enter without +1/+1

**Comment by awjackson** [comment]
This is not a bug, it's very specifically what is supposed to happen according to the rules:

>707.9e Some replacement effects that generate copy effects include an exception that’s an additional effect rather than a modification of the affected object’s characteristics. If another copy effect is applied to that object after applying the copy effect with that exception, the exception’s effect doesn’t happen. Example: Altered Ego reads, “You may have Altered Ego enter the battlefield as a copy of any creature on the battlefield, except it enters with X additional +1/+1 counters on it.” You choose for it to enter the battlefield as a copy of Clone, which reads “You may have Clone enter the battlefield as a copy of any creature on the battlefield,” for which no creature was chosen as it entered the battlefield. If you then choose a creature to copy as you apply the replacement effect Altered Ego gains by copying Clone, Altered Ego’s replacement effect won’t cause it to enter the battlefield with any +1/+1 counters on it.

**Comment by awjackson** [comment]
On second thought, I think this is an actual bug. The example in the CR is Altered Ego copying a Clone *for which no creature was chosen* (so you apply the Clone's ETB replacement effect after applying Alter Ego's) In this case a creature *was* chosen for the Clone, so you aren't applying a second copy replacement effect.

**Comment by awjackson** [comment]
If this is a bug, the problem is with the way we apply "enters with an additional counter" by splicing an ability onto the copy. Entering with a counter shouldn't be a gained ability--if it were, it would be worded 'it has "this creature enters the battlefield with a counter"'.

**Comment by copperfield42** [comment]
yes, I also ask in the judge chat and they said that it should enter with counter

**Comment by xenohedron** [comment]
Related to #10801
